tests/functional/aws-node-sdk/lib/utility/createEncryptedBucket.js

```diff
@@ -1,5 +1,5 @@
 const childProcess = require('child_process');
-const Promise = require('bluebird');
+const myPromise = require('bluebird');
```

Comment:
rename `bluebirdPromise` (or `bbPromise`).... or can we switch to using "standard" node.js Promise instead ?